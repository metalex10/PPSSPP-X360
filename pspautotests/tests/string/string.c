#include <common.h>

#include <pspkernel.h>
#include <pspthreadman.h>
#include <stdio.h>
#include <string.h>

float value = 13.0;

int main(int argc, char** argv) {
	char buffer[128];
	
	sprintf(buffer, "%u", (uint)0);
	printf("%s\n", buffer);
	
	sprintf(buffer, "%u", (uint)100000);
	printf("%s\n", buffer);

	sprintf(buffer, "%llu", (u64)9);
	printf("%s\n", buffer);

	sprintf(buffer, "%llu", (u64)100000);
	printf("%s\n", buffer);

	sprintf(buffer, "%.2f", value);
	printf("%s\n", buffer);

	return 0;
}
