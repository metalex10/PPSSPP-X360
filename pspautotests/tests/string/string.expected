0
100000
9
100000
13.00
